packages/x-charts-pro/src/ChartContainerPro/ChartContainerPro.tsx

```diff
@@ -0,0 +1,358 @@
+import * as React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ChartContainerProps,
+  ChartsSurface,
+  HighlightedProvider,
+  ZAxisContextProvider,
+} from '@mui/x-charts';
+import {
+  CartesianContextProvider,
+  ChartsAxesGradients,
+  ColorProvider,
+  DrawingProvider,
+  InteractionProvider,
+  SeriesContextProvider,
+  useChartContainerHooks,
+} from '@mui/x-charts/internals';
+
+export type ChartContainerProProps = ChartContainerProps;
```

Comment:
```suggestion
export interface ChartContainerProProps extends ChartContainerProps {}
```

Keeping it as an interface has some small benefits (module declaration, can expend this interface elsewhere if needed, ...)